test/specs/commands/createTransactionCastVotes.js

```diff
@@ -809,6 +809,28 @@ describe('create transaction cast votes', () => {
 								);
 							},
 						);
+						Given(
+							'a public key that should be voted for "1234567812345678123456781234567812345678123456781234567812345678"',
+							given.aPublicKeyThatShouldBeVotedFor,
+							() => {
+								Given(
+									'an options object with signature set to false and votes set to "1234567812345678123456781234567812345678123456781234567812345678" ',
+									given.anOptionsObjectWithSignatureSetToAndVotesSetTo,
+									() => {
+										When(
+											'the action is called with the options',
+											when.theActionIsCalledWithTheOptions,
+											() => {
+												Then(
+													'it should resolve to the created transaction',
```

Comment:
I think we still need something similar to what was here before but relevant to the particular case. E.g. `it should create a cast votes transaction with the public keys prepended with a plus`